testing/nginx-naxsi/APKBUILD

```diff
@@ -32,119 +51,154 @@ source="http://nginx.org/download/$_pkgname-$pkgver.tar.gz
+                --http-client-body-temp-path=/var/lib/$_pkgname/tmp/client_body \
+                --http-proxy-temp-path=/var/lib/$_pkgname/tmp/proxy \
+                --http-fastcgi-temp-path=/var/lib/$_pkgname/tmp/fastcgi \
+                --with-perl_modules_path=/usr/lib/perl5/vendor_perl \
+                \
+                --user=$pkgusers \
+                --group=$_grp_ngx \
+                --with-threads \
+                --with-file-aio \
+                --without-http_uwsgi_module \
+                --without-http_scgi_module \
+                \
+                --with-http_ssl_module \
+                --with-http_v2_module \
+                --with-http_realip_module \
+                --with-http_addition_module \
+                --with-http_sub_module \
+                --with-http_dav_module \
+                --with-http_flv_module \
+                --with-http_mp4_module \
+                --with-http_gunzip_module \
+                --with-http_gzip_static_module \
+                --with-http_auth_request_module \
+                --with-http_random_index_module \
+                --with-http_secure_link_module \
+                --with-http_slice_module \
+                --with-http_stub_status_module \
+                --with-http_realip_module \
+                --with-http_xslt_module=dynamic \
+                --with-http_image_filter_module=dynamic \
+                --with-http_geoip_module=dynamic \
+                --with-http_perl_module=dynamic \
+                --with-mail=dynamic \
+                --with-mail_ssl_module \
+                --with-stream=dynamic \
+                --with-stream_ssl_module \
+                \
+                --add-dynamic-module="$_ngx_naxsi_dir" \
+                --add-dynamic-module="$_ngx_cache_purge_dir" \
+                --add-dynamic-module="$_ngx_upstream_fair_dir" \
+                --add-dynamic-module="$_ngx_http_sysguard_dir" \
 		|| return 1
 	make || return 1
 }
 
 package() {
-	cd "$_builddir"
+	cd "$builddir"
 	make DESTDIR="$pkgdir" install
 
-	local paxflags="-m"
-	[ "$CARCH" = "x86" ] && paxflags="-msp"
-	paxmark "$paxflags" "$pkgdir"/usr/sbin/nginx || return 1
+	# Disable some PaX protections; this is needed for Lua module.
+#	local paxflags="-m"
+#	[ "$CARCH" = "x86" ] && paxflags="-msp"
+#	paxmark "$paxflags" "$pkgdir"/usr/sbin/nginx || return 1
 
 	install -m755 -D "$srcdir"/$_pkgname.initd "$pkgdir"/etc/init.d/$_pkgname
 	install -m644 -D "$srcdir"/$_pkgname.logrotate "$pkgdir"/etc/logrotate.d/$_pkgname
 
 	install -m644 -D LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 	install -m644 -D "$srcdir"/naxsi-$_ngx_naxsi_ver/naxsi_config/naxsi_core.rules "$pkgdir"/etc/nginx/naxsi_core.rules
+	mkdir -p "$pkgdir"/var/lib/nginx/tmp
 }
 
 vim() {
-        local t
+        pkgdesc="$pkgdesc (vim syntax)"
+        depends=
+
+        mkdir -p "$subpkgdir"/usr/share/vim
+        cp -r "$builddir"/contrib/vim "$subpkgdir"/usr/share/vim/vimfiles
+}
```

Comment:
There’s already `nginx-vim` provided by the `main/nginx` package, so this subpackage is redundant.